src/App.tsx

```diff
@@ -1,18 +1,43 @@
 import React from 'react';
 
 import './App.scss';
+import { TabsList } from './components/TabsList';
 
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type State = {
+  ourTabs: Tab[],
+  selectedTab: Tab,
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    ourTabs: [...tabs],
```

Comment:
You don't need to store tabs in the state
```suggestion
    ourTabs: [...tabs],
```